java/code/src/com/suse/manager/webui/controllers/SystemsController.java

```diff
@@ -321,6 +323,17 @@ public static Object getAccessibleChannelChildren(Request request, Response resp
                         HttpStatus.SC_BAD_REQUEST,
                         ResultJson.error("invalid_channel_id"));
             }
+
+            Channel oldBaseChannel =  null;
+            try {
+                Long oldBaseChannelId = Long.parseLong(request.queryParams("oldBaseChannelId"));
+                oldBaseChannel = ChannelManager.lookupByIdAndUser(oldBaseChannelId, user);
+            }
+            catch (Exception e) {
+                //This is not a critical feature for this request, we want to keep move forward anyway
+                LOG.debug("An oldBaseChannelId parameter was provided when fetching childchannels");
```

Comment:
Is here a "not" missing? I do not understand the log message.

Reply:
yes! A not was missing here, thanks.

I wonder if this LOG.debug shouldn't be an error :thinking: :thinking: 

If there is a bug on the frontend and this parameter oldBaseChannelId isn't provided, we will never be aware that something is wrong with the pre-selection. Should we consider this as critical and abort the request with an error instead of providing the child channels anyway with the pre selection info off?

Reply:
Well, I am not sure if you hit this if the system does not have a channel. This can happen for Bootstrap entitled systems.
But "info" or "warn" would be good.
(Info is not logged by default, while warn is)

Reply:
hum ok,
I changed it to log only if the query parameter oldBaseChannelId is provided and something unexpected happen